locales/it/index.yml

```
@@ -1,6 +1,8 @@
 ---
 global:
   accessibility:
+    expanded: Espanso
```

```suggestion
    expanded: Paragrafo espanso
```
Could it be clearer we have a content that could be closed/opened?
Perhaps `Sezione` could be better?

---

Maybe Paragraph better explains it has an expanded text below, I'll apply this changes in the next commit

---

@Undermaken updated in 3f1674b